src/commands/object/delete.ts

```diff
@@ -33,26 +30,12 @@ export class DeleteCommand extends Command {
                 console.log();
             }
         } else {
-            const obj = session.getObject<graphene.Storage>(Handle.toBuffer(params.oid));
-            if (!obj) {
-                throw new Error(`Object by ID '${params.oid}' is not found`);
-            }
-
-            // Print info about object
-            print_caption(`Object info`);
-            print_object_info(obj);
-            console.log();
-
-            const answer = await readline.question("Do you really want to remove this object (Y/N)? ");
```

Comment:
Reminder: this should not have been removed.
Also should not prompt in non-interactive mode.